strawberry/experimental/pydantic/conversion.py

```diff
@@ -87,7 +89,7 @@ def convert_pydantic_model_to_strawberry_class(cls, *, model_instance=None, extr
     return cls(**kwargs)
 
 
-def convert_strawberry_class_to_pydantic_model(obj):
+def convert_strawberry_class_to_pydantic_model(obj) -> Any:
```

Comment:
@thejaminator does this type make sense?

Reply:
yes, the function name seems abit of a misnomer, but its more like recursively converting the fields of a strawberry type into the fields of a pydantic model.

which can be anything